packages/babel-plugin-remove-graphql-queries/src/__tests__/__snapshots__/index.js.snap

```diff
@@ -67,6 +87,34 @@ export default (() => React.createElement(StaticQuery, {
 
 exports[`Transforms queries in page components 1`] = `"export const query = \\"3687030656\\";"`;
 
+exports[`Transforms queries in useStaticQuery 1`] = `
+"import staticQueryData from \\"public/static/d/2626356014.json\\";
+import React from 'react';
+import { useStaticQuery } from 'gatsby';
+export default (() => {
+  const siteTitle = staticQueryData.data;
+  return React.createElement(\\"h1\\", null, siteTitle.site.siteMetadata.title);
+});"
+`;
+
+exports[`Works correctly with the kitchen sink 1`] = `
```

Comment:
@pieh So this is _pretty_ broken but not sure because of _which_ transform

What should the correct result be here?

Reply:
So a correct result (I think) would be something like (largely pseudo code)

```jsx
import staticQueryData1 from 'public/static/d/unique-hash-1.json'
import staticQueryData2 from 'public/static/d/unique-hash-2.json'
import React from 'react'
import { useStaticQuery, StaticQuery } from 'gatsby'

export default () => {
  const query = 'unique-hash-1'
  const siteDescription = staticQueryData1.data

  return (
    React.createElement(StaticQuery, { /* yadda */}, {
      data: staticQueryData2
    })
  )
}
```